etc/answers-core.api.md

```diff
@@ -287,6 +285,21 @@ export interface NearFilterValue {
     radius: number;
 }
 
+// @public
+export interface NumberRangeLimit {
+    matcher: NumberRangeMatcher;
+    value: number;
+}
+
+// @public
+export type NumberRangeMatcher = Matcher.GreaterThan | Matcher.GreaterThanOrEqualTo | Matcher.LessThan | Matcher.LessThanOrEqualTo;
+
+// @public
+export interface NumberRangeValue {
+    end: NumberRangeLimit;

```

Comment:
What are your thoughts on creating a `LowerNumberRange` and a `UpperNumberRange` model? That way we can statically enforce that nobody creates a `NumberRangeValue` with two `GreaterThan` or two `LessThan` matchers